storybook/preview.ts

```diff
@@ -22,6 +22,14 @@ const preview: Preview = {
     },
     // Sets a delay of 1550ms before each story to give them time to animate (recharts animation default is 1500ms)
     chromatic: { delay: 1550 },
+    codesandbox: {
+      apiToken: '',
```

Comment:
So I would like to use this addon but I can't see how to get this token through. It's private so we cannot just embed it in the source code. Thoughts?

Reply:
I changed you to Admin in codesandbox - go to recharts -> settings -> permissions -> API Access